packages/now-build-utils/src/detectors/gatsby.ts

```diff
@@ -0,0 +1,19 @@
+import { DetectorParameters, DetectorResult } from '../types';
+
+export default async function detectGatsby({
+  fs: { exists, hasDependency },
+}: DetectorParameters): Promise<DetectorResult> {
+  const hasGatsby = await hasDependency('gatsby');
+  if (!hasGatsby) {
+    return false;
+  }
+  const hasConfig = await exists('gatsby-config.js');
+  if (!hasConfig) {
```

Comment:
Why is `gatsby-config.js` required?

Reply:
You think that checking for only `gatsby` dependency is good enough? We could probably remove this.